assets/css/src/global.css

```diff
@@ -1,3 +1,28 @@
+/*
+CSS normalization based in part on normalize.css by
```

Comment:
We should probably keep this comment in `reset.css`

Reply:
Yes